### PR TITLE
Cleanups and fixes for LDAP section of Security guide

### PR DESCRIPTION
**Pull request by cjschroder**
Removed references to YaST LDAP/Kerberos  module, clarified INF file usage,
some other small corrections from feedback to https://github.com/SUSE/doc-sle/pull/711

### Are there any relevant issues/feature requests?

https://jira.suse.com/browse/SLE-16707
https://jira.suse.com/browse/SLE-14732

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ---------- 
| [x ] 15 SP3  | (`master` only)   
| [ ] 15 SP2  | [ ] 
| [ ] 15 SP1  | [ ] 
| [ ] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

**Comment by cjschroder** [comment]
@Firstyear please take a look.

**Review by Firstyear — APPROVED** [review]
All seems reasonable to me!

**Comment by cjschroder** [comment]
eeeks I pushed this too soon & left out some stashed changes. If you see anything weird let me know. Consider this a work in progress, with each commit (hopefully) detailing correct procedures, and subject to change in the next commit with improvements and corrections. I will push new material quickly, until it reaches a reasonable state of usefulness and not-wrong-outdated.

File: xml/security_ldap.xml [hunk 1]
```diff
@@ -320,15 +320,32 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
  <sect1 xml:id="sec-security-ldap-server">
   <title>Setting up a &ds389; Testing and Development Instance</title>
   <para>
-   Follow these steps to set up a simple instance for testing and development, 
-   populated with a small set of sample entries.
+      You will use the <command>dscreate</command> command to create and remove
```
Comment (Firstyear) [inline xml/security_ldap.xml:323]: Only to create, dsctl removes :)

File: xml/security_ldap.xml [hunk 2]
```diff
@@ -398,15 +415,13 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
...
+    <sect2 xml:id="sec-security-ldap-server-instance">
   <title>Creating a &ds389; Instance with a Custom Configuration File</title>
     <para>
-     Use the <command>dscreate</command> command to create a new &ds389; instance.
-     There are several ways to create a new instance: interactively, from a 
-     template file, and using a custom configuration file.
-     In this section you will learn how to use a custom configuration file. This 
-     file must be in the INF format. You may name it anything you like.
+     You may create a new &ds389; instance from a simple custom configuration
+     file. This file must be in the INF format, and you may name it anything you
```
Comment (Firstyear) [inline xml/security_ldap.xml:423]: 👻.🍿.inf

Comment (cjschroder) [reply xml/security_ldap.xml:423]: :beers:

Comment (cjschroder) [reply xml/security_ldap.xml:423]: INF is correct, it is the name of the file format. The .inf extension is not required.

fixed dscreate/dsctl error https://github.com/SUSE/doc-sle/pull/732

Comment (Firstyear) [reply xml/security_ldap.xml:423]: inf.✨

**Pull request merged**